src/lib/components/Modal/Modal.jsx

```diff
@@ -97,9 +97,14 @@ export const Modal = ({
   };
 
   useEffect(() => {
+    const bodyElement = document.getElementsByTagName('body')[0];
+    bodyElement.style = 'overflow-y: hidden';
```

Comment:
How about:

```suggestion
    window.document.body.style.overflowY = 'hidden';
```

(`window` is optional, but let's keep it the same as below)